datafusion/physical-expr/Cargo.toml

```diff
@@ -44,6 +44,7 @@ arrow = { version = "17.0.0", features = ["prettyprint"] }
 blake2 = { version = "^0.10.2", optional = true }
 blake3 = { version = "1.0", optional = true }
 chrono = { version = "0.4", default-features = false }
+chronoutil = "0.2.3"
```

Comment:
I am somewhat concerned about adding dependencies (and this particular crate seems to be maintained by a single person) -- however,  since it is so small (and MIT licensced), we could also just inline the functions we care about.

Any thoughts @andygrove or @thinkharderdev ?

https://github.com/olliemath/chronoutil/blob/main/src/delta.rs

Reply:
I haven't reviewed the PR so am not sure how much of that create is used but I agree if it is a small amount of code it is better to just copy the code rather than add another dependency.

Reply:
I inlined it in my latest push. If we want to go back to including it in cargo.toml, it's an easy revert.

My $0.02 would be to leave it in cargo.toml, so that we can get updates if the author fixes something, and so that RAT checkers can see declaratively that we depend on that code, rather than trying to use heuristics of sematic code or binary diff.